assets/dev/scss/frontend/widgets/link-in-bio.scss

```diff
@@ -0,0 +1,93 @@
+.elementor-widget-link-in-bio {
+	--align-self: center;
+	--e-link-in-bio-gutter: 2em;
+
+	.elementor-widget-container {
+		border: 1px solid blue;
+		margin-inline: auto;
+		max-width: 100%;
+		padding: var(--e-link-in-bio-gutter);
+		padding-block-start: 0;
+		width: 100%;
+	}
+}
+
+.e-link-in-bio {
+
+	&__content-container {
+		display: flex;
+		flex-direction: column;
+		gap: 1em;
+		margin-inline: auto;
+		text-align: center;
+
+		* {
+			word-wrap: break-word;
+		}
+	}
+
+	&__identity--profile {
+		padding-block-start: var(--e-link-in-bio-gutter);
+
+		img.e-link-in-bio__identity-image {
+			aspect-ratio: 1;
+			border: 1px solid blue;
+			border-radius: 50%;
+			display: block;
+			object-fit: cover;
+			margin-inline: auto;
+			max-width: 100%;
+			width: 150px;
+		}
+	}
+
+	&__identity--cover {
+
+		img.e-link-in-bio__identity-image {
```

Comment:
@rodolphebertozzo @louiswol94 the use if img here was to add specificity required to override core styles.
The CSS in this PR is very much just to support the initial HTML work and has since been refined and will be coming in a later PR including this instances where a more aligned approach was applied.

Reply:
@willhowat Noted, thanks. We normally don't use the html tags in our selectors, so let me know if there is an issue here then we can look at somehow making your selector more specific.

Reply:
@louiswol94 no issue just needed a specificity bump through nesting selectors instead. This is resolved in the latest updates to this PR